src/components/app/OverlayField.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import MasterWidget from '../widget/MasterWidget';
+
+import {
+    findRowByPropName
+} from '../../actions/WindowActions';
+
+class OverlayField extends Component {
+
```

Comment:
Oh my god ;p How it is working without actually passing props to constructor of parent class? super(props) is missing!